test/CSSTransition-test.js

```diff
@@ -95,19 +95,22 @@ describe('CSSTransition', () => {
       instance.props({
         in: true,
 
-        onEnter(node){
+        onEnter(node, appearing){
           count++;
           expect(node.className).toEqual('custom');
+          expect(appearing).toBe(false);
         },
 
-        onEntering(node){
+        onEntering(node, appearing){
           count++;
           expect(node.className).toEqual('custom custom-super-active');
+          expect(appearing).toBe(false);
         },
 
-        onEntered(node){
+        onEntered(node, appearing){
           expect(node.className).toEqual('');
           expect(count).toEqual(2);
+          expect(appearing).toBe(false);
```

Comment:
I gave it a try but I'm not sure how to do it simple or how to do it at all. I tried to find an example for mounting a component but couldn't find on that file. I tried to do something like this but it didn't work:
```

    it.only('should apply pass isAppearing true', () => {

      let appearing = {}

      jest.resetModuleRegistry();
      jest.useFakeTimers();

      const container = document.createElement('div');
      const ReactDOM = require('react-dom');
      class Yolo extends React.Component {

        onEnter(node, isAppearing) {
          appearing.onEnter = isAppearing;
        }

        render() {
          return (
            <CSSTransition classNames="yolo" appear={true} enter={true} timeout={10} onEnter={this.onEnter}>
              <span>Spanning around</span>
            </CSSTransition>
          )
        }
      }

      ReactDOM.render(
        <Yolo/>,
        container,
      );

      jest.runAllTimers();

      expect(appearing.onEnter).toBe(true); // returns undefined === onEnter wasn't even called

    });
```